scripts/linters/python_linter.py

```diff
@@ -42,12 +42,37 @@
 # pylint: disable=wrong-import-order
 # pylint: disable=wrong-import-position
 from pylint import lint  # isort:skip
+from pylint.reporters import text# isort:skip
 import isort  # isort:skip
 import pycodestyle # isort:skip
 # pylint: enable=wrong-import-order
 # pylint: enable=wrong-import-position
 
 
+class StringMessageStream(python_utils.OBJECT):
+    """Save and return output stream of pylint."""
```

Comment:
Save and return an output stream.

Also, for the docstrings for write() and read(), rephrase them to avoid mention of pylint.

Reply:
Done.